bridge/client/app/_interceptors/http-error-interceptor.spec.ts

```diff
@@ -118,4 +118,86 @@ describe('HttpErrorInterceptorService', () => {
     // then
     expect(spy).toHaveBeenCalledTimes(0);
   });
+
+  it('should show error only once in case of 401 and OAUTH', () => {
+    // given
+    const spy = jest.spyOn(TestBed.inject(NotificationsService), 'addNotification');
+
+    apiService.getMetadata().subscribe();
+    apiService.getProjects().subscribe();
+
+    const testRequest: TestRequest = httpMock.expectOne('./api/v1/metadata');
+    const errorEvent: ErrorEvent = new ErrorEvent('', { error: {} });
+    const headers = new HttpHeaders({ 'keptn-auth-type': 'OAUTH' });
+    testRequest.error(errorEvent, { headers, status: 401 });
+
+    const testRequestProjects: TestRequest = httpMock.expectOne(
+      './api/controlPlane/v1/project?disableUpstreamSync=true'
+    );
+    testRequestProjects.error(errorEvent, { headers, status: 401 });
+
+    // then
+    expect(spy).toBeCalledTimes(1);
+    expect(spy).toHaveBeenCalledWith(NotificationType.INFO, 'Login required. Redirecting to login.');
+  });
+
+  it('should show error only once in case of 401 and BASIC auth', () => {
+    // given
+    const spy = jest.spyOn(TestBed.inject(NotificationsService), 'addNotification');
+
+    apiService.getMetadata().subscribe();
+    apiService.getProjects().subscribe();
+
+    const testRequest: TestRequest = httpMock.expectOne('./api/v1/metadata');
+    const errorEvent: ErrorEvent = new ErrorEvent('', { error: {} });
+    const headers = new HttpHeaders({ 'keptn-auth-type': 'BASIC' });
+    testRequest.error(errorEvent, { headers, status: 401 });
+
+    const testRequestProjects: TestRequest = httpMock.expectOne(
+      './api/controlPlane/v1/project?disableUpstreamSync=true'
+    );
+    testRequestProjects.error(errorEvent, { headers, status: 401 });
+
+    // then
+    expect(spy).toBeCalledTimes(1);
+    expect(spy).toHaveBeenCalledWith(
+      NotificationType.ERROR,
+      'Login credentials invalid. Please check your provided username and password.'
+    );
+  });
+
+  it('should show error only once in case of 401', () => {
+    // given
+    const spy = jest.spyOn(TestBed.inject(NotificationsService), 'addNotification');
+
+    apiService.getMetadata().subscribe();
+    apiService.getProjects().subscribe();
+
+    const testRequest: TestRequest = httpMock.expectOne('./api/v1/metadata');
+    const errorEvent: ErrorEvent = new ErrorEvent('incorrect api key auth');
+    testRequest.error(errorEvent, { status: 401 });
+
+    const testRequestProjects: TestRequest = httpMock.expectOne(
+      './api/controlPlane/v1/project?disableUpstreamSync=true'
+    );
+    testRequestProjects.error(errorEvent, { status: 401 });
+
+    // then
+    expect(spy).toBeCalledTimes(1);
+    expect(spy).toHaveBeenCalledWith(NotificationType.ERROR, 'Could not authorize.');
```

Comment:
With `ErrorEvent('incorrect api key auth');` shouldn't this lead to `Could not authorize API token. Please check the configured API token.`?